test/expected/alter.out

```diff
@@ -194,3 +194,136 @@ SELECT * FROM _timescaledb_catalog.chunk WHERE id = 2;
 ALTER TABLE public.new_chunk_name RENAME COLUMN time TO newtime;
 ERROR:  cannot rename column "time" of hypertable chunk "new_chunk_name"
 \set ON_ERROR_STOP 1
+\! mkdir /tmp/new_space
+CREATE TABLESPACE new_space LOCATION '/tmp/new_space';
+CREATE TABLE hyper_in_space(time bigint, temp float, device int);
+SELECT create_hypertable('hyper_in_space', 'time', 'device', 4, chunk_time_interval=>1);
+NOTICE:  adding not-null constraint to column "time"
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+INSERT INTO hyper_in_space(time, temp, device) VALUES (1, 20, 1);
+INSERT INTO hyper_in_space(time, temp, device) VALUES (3, 21, 2);
+INSERT INTO hyper_in_space(time, temp, device) VALUES (5, 23, 1);
+SELECT tablename FROM pg_tables WHERE tablespace = 'new_space';
+ tablename 
+-----------
+(0 rows)
+
+ALTER TABLE hyper_in_space SET TABLESPACE new_space;
+SELECT tablename FROM pg_tables WHERE tablespace = 'new_space';
+    tablename     
+------------------
+ hyper_in_space
+ _hyper_3_5_chunk
+ _hyper_3_6_chunk
+ _hyper_3_7_chunk
+(4 rows)
+
+-- should be inserted in an existing chunk in the new tablespace,
+-- no new chunks
+INSERT INTO hyper_in_space(time, temp, device) VALUES (5, 27, 1);
+-- should be inserted in the default tablespace
+INSERT INTO hyper_in_space(time, temp, device) VALUES (8, 24, 2);
+SELECT tablename, tablespace FROM pg_tables
+WHERE tablename = 'hyper_in_space' OR tablename LIKE '\_hyper\__\__\_chunk' ORDER BY tablename;
+    tablename     | tablespace 
+------------------+------------
+ _hyper_1_1_chunk | 
+ _hyper_2_3_chunk | 
+ _hyper_2_4_chunk | 
+ _hyper_3_5_chunk | new_space
+ _hyper_3_6_chunk | new_space
+ _hyper_3_7_chunk | new_space
+ _hyper_3_8_chunk | 
```

Comment:
This is odd, I'd expect new chunks to be created in the hypertable's tablespace.